core/state/database.go

```diff
@@ -34,10 +34,10 @@ import (
 
 const (
 	// Number of codehash->size associations to keep.
-	codeSizeCacheSize = 100000
+	codeSizeCacheSize = 1000_000 // 4 megabytes in total
```

Comment:
You increased the code cache size by 10x here

Reply:
yes, but the total size is only 4mb.

Reply:
Should be formated like this though imo
```suggestion
	codeSizeCacheSize = 1_000_000 // 4 megabytes in total
```